flow/record/fieldtypes/net/ip.py

```diff
@@ -8,49 +11,55 @@ class ipaddress(FieldType):
     val = None
     _type = "net.ipaddress"
 
-    def __init__(self, addr):
+    def __init__(self, addr: Union[str, int]):
```

Comment:
Actually ipaddress can also initialize from 4 bytes, and can you change it to new style typing? eg: `str | int | bytes`

https://docs.python.org/3/library/ipaddress.html#ipaddress.IPv4Address